bridge/src/androidTest/java/com/livefront/bridge/disk/FileDiskHandlerTest.java

```diff
@@ -118,6 +118,13 @@ public void constructor() {
         assertNull(fileDiskHandler3.getBytes(key));
     }
 
+    @Test
+    public void getBytes_afterCancelling() {
+        // Should not trigger a CancellationException
+        mFileDiskHandler.clearAll();
+        mFileDiskHandler.getBytes("test");
```

Comment:
So this would crash without your fix?

Reply:
Yup. And all I can really do here is write a test that _doesn't crash_ after the fix.